src/platform/unittest/TlsTest.cpp

```diff
@@ -64,6 +115,19 @@ struct TlsTest : public ::testing::TestWithParam<bool>
         ClientCertParams = (QUIC_CREDENTIAL_CONFIG*)CxPlatGetSelfSignedCert(CXPLAT_SELF_SIGN_CERT_USER, TRUE);
         ASSERT_NE(nullptr, ClientCertParams);
         ClientCertParams->Flags |= QUIC_CREDENTIAL_FLAG_NO_CERTIFICATE_VALIDATION;
+#endif
+#ifndef QUIC_DISABLE_PFX_TESTS
+        CertParamsFromFile = (QUIC_CREDENTIAL_CONFIG*)CXPLAT_ALLOC_NONPAGED(sizeof(QUIC_CREDENTIAL_CONFIG), QUIC_POOL_TEST);
+        ASSERT_NE(nullptr, CertParamsFromFile);
+        CxPlatZeroMemory(CertParamsFromFile, sizeof(*CertParamsFromFile));
+        CertParamsFromFile->Type = QUIC_CREDENTIAL_TYPE_CERTIFICATE_PKCS12;
+        CertParamsFromFile->CertificatePkcs12 = (QUIC_CERTIFICATE_PKCS12*)CXPLAT_ALLOC_NONPAGED(sizeof(QUIC_CERTIFICATE_PKCS12), QUIC_POOL_TEST);
+        ASSERT_NE(nullptr, CertParamsFromFile->CertificatePkcs12);
+        CxPlatZeroMemory(CertParamsFromFile->CertificatePkcs12, sizeof(QUIC_CERTIFICATE_PKCS12));
+        CertParamsFromFile->CertificatePkcs12->Asn1Blob = (const uint8_t *) ReadFile(TestPath, &CertParamsFromFile->CertificatePkcs12->Asn1BlobLength);
+        CertParamsFromFile->CertificatePkcs12->PrivateKeyPassword = PfxPass;
+        ASSERT_NE(nullptr, CertParamsFromFile->CertificatePkcs12->Asn1Blob);
+        ASSERT_NE(0, CertParamsFromFile->CertificatePkcs12->Asn1BlobLength);
```

Comment:
Looks like the compiler is complaining about this. Try this?
```suggestion
        ASSERT_NE((uint32_t)0, CertParamsFromFile->CertificatePkcs12->Asn1BlobLength);
```